### DISCUSSION TITLE
Change amount of forms through input from another form

### DISCUSSION THREAD

**Discussion by jahknem** [Budibase/budibase#5189] [Help]
Hi! I'm trying to create a webform in which a support technician can enter customer devices as well as their MAC Adresses. It is possible for the customer to have an arbitrary amount of devices. Due to this I would like to enable the support technician to change the number of possible device entrys on the fly. 
Currently I have this form:
![image](https://user-images.githubusercontent.com/33702431/161037599-5fa0e07e-036f-4df5-b800-7494bb8b6d19.png)

What I would like is the second line (Choose an option, Text Entry, Button) to be added every time I click the "add device" button. Is there any way to do that? I'm also thinking about adding a number field beside the button to be able to add multiple entries at the same time to the form.

**Comment by melohagan** [comment]
Hi @jahknem 

I explored some options with App State, Repeaters, JavaScript and JSON Columns. 

I think at the moment achieving exactly what you are looking for isn't possible, but it has highlighted an area for improvement, so thank you for post!
